rosetta/examples/create_transaction.go

```diff
@@ -72,9 +72,10 @@ func main() {
 	}
 
 	mtx := rosettaFilecoinLib.TxMetadata{
-		Nonce:    uint64(respMetadata.Metadata[services.NonceKey].(float64)),
-		GasPrice: uint64(respMetadata.Metadata[services.GasPriceKey].(float64)),
-		GasLimit: gasLimit,
+		Nonce:      uint64(respMetadata.Metadata[services.NonceKey].(float64)),
+		GasPremium: uint64(respMetadata.Metadata[services.GasPremiumKey].(float64)),
```

Comment:
We will have a problem here..
`GasPremium` and `GasFeeCap` are `big.Int` so float64 may not be enough to represent the full value.
I think we should work with strings here instead

Reply:
Ok, fixed it, this was outdated since gas data comes originally on string format.